### PR TITLE
Add @active-item-bg to control hover and active states

### PR DESCRIPTION
**Pull request by megawac**
`@primary-color` is used for a couple distinct purposes (alert backgrounds, hover/active backgrounds, calendar colors). I've added a variable to help control the appearance of hover and active states. 

I've found I wanted to control alert info backgrounds seperately from hover/active states and `@primary-1` seemed to make more sense for the `-info` background

### CODE REVIEW

File: components/cascader/style/index.less [hunk 1]
```diff
@@ -174,7 +174,7 @@
     &-active:not(&-disabled) {
       &,
       &:hover {
-        background: @background-color-base;
+        background: @background-color-active;
```
Comment (afc163) [inline components/cascader/style/index.less]: Don't change that, it should not be a active background.

**Comment by afc163** [comment]
Active is more like a pressed state, so I don't think `@active-item-bg` is good idea.

**Comment by afc163** [comment]
`@active-item-bg` => `@item-hover-bg`

**Comment by megawac** [comment]
@afc163 how about we seperate this variable into another 2:

- `@active-item-bg`
- `@hover-item-bg`

**Comment by megawac** [comment]
@afc163 take another look when you're online :)

**Pull request merged**